templates/components/styles/global_nav.html.twig

```diff
@@ -11,7 +13,25 @@
         <ul data-component="nav-double-intro">
             {%~ for navItem in navigation %}
                 <li class="top-nav-item has-children">
-                    <a href="{{ navItem.titleLink }}" class="nav-link">{{ navItem.title }}</a>
+                        {% if navItem.targetLinkLanguage is not empty %}
+                            {% set targetLangLabel = '' %}
+                            {% set langDiff = false %}
+                            {% if navItem.targetLinkLanguage is not same as siteLang %}
+                                {% set langDiff = true %}
+
+                                {% if navItem.targetLinkLanguage == 'zh-Hans' %}
+                                    {% set targetLangLabel = "Chinese" %}
+                                {% else %}
+                                    {% set targetLangLabel = "English" %}
+                                {% endif %}
+                            {% endif %}
+                        {% endif %}
+                    <a href="{{ navItem.titleLink }}" class="nav-link"{% if langDiff is defined and langDiff %} hreflang="{{ navItem.targetLinkLanguage }}"{% endif %}>
+                        {{ navItem.title }}
+                        {% if navItem.targetLinkLanguage is not empty %}
+                            {{ (langDiff is defined and langDiff) ? '(' ~ targetLangLabel ~ ')' : '' }}
```

Comment:
When the target language is different from the current one, you chose to display it as "(Language)". Do you think it should be styled to be less prominent?

https://www.w3.org/International/questions/qa-link-lang suggests some CSS, that, despite not being accessible (at least because of the contrast ratio), makes that less "in your face". It also suggests adding the language through CSS content rather than in the markup, but I don't know what are the implications of one or the other.